homeassistant/components/airnow/sensor.py

```diff
@@ -0,0 +1,125 @@
+"""Support for the Airly sensor service."""
+from homeassistant.const import (
+    ATTR_ATTRIBUTION,
+    ATTR_DEVICE_CLASS,
+    CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
+    CONCENTRATION_PARTS_PER_MILLION,
+    CONF_NAME,
+)
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import (
+    ATTR_API_AQI,
+    ATTR_API_AQI_DESCRIPTION,
+    ATTR_API_AQI_LEVEL,
+    ATTR_API_O3,
+    ATTR_API_PM25,
+    DOMAIN,
+    SENSOR_AQI_ATTR_DESCR,
+    SENSOR_AQI_ATTR_LEVEL,
+)
+
+ATTRIBUTION = "Data provided by AirNow"
+
+ATTR_ICON = "icon"
+ATTR_LABEL = "label"
+ATTR_UNIT = "unit"
+
+PARALLEL_UPDATES = 1
+
+SENSOR_TYPES = {
+    ATTR_API_AQI: {
+        ATTR_DEVICE_CLASS: None,
+        ATTR_ICON: "mdi:blur",
+        ATTR_LABEL: ATTR_API_AQI,
+        ATTR_UNIT: "aqi",
+    },
+    ATTR_API_PM25: {
+        ATTR_DEVICE_CLASS: None,
+        ATTR_ICON: "mdi:blur",
+        ATTR_LABEL: ATTR_API_PM25,
+        ATTR_UNIT: CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
+    },
+    ATTR_API_O3: {
+        ATTR_DEVICE_CLASS: None,
+        ATTR_ICON: "mdi:blur",
+        ATTR_LABEL: ATTR_API_O3,
+        ATTR_UNIT: CONCENTRATION_PARTS_PER_MILLION,
+    },
+}
+
+
+async def async_setup_entry(hass, config_entry, async_add_entities):
+    """Set up Airly sensor entities based on a config entry."""
```

Comment:
AirNow not Airly 😉

Reply:
Oops - now everyone knows where I copied from :) Fixed in f426c71